@here/harp-datasource-protocol/lib/DecodedTile.ts

```diff
@@ -87,9 +87,10 @@ export interface PathGeometry {
 
 /**
  * Attributes corresponding to some decoded geometry. It may be either a map
- * of multiple attributes or just a number with the geometry's feature id.
+ * of multiple attributes or just a string with the geometry's feature id (id numbers are
+ * deprecated).
  */
-export type AttributeMap = {} | number;
+export type AttributeMap = {} | string | number;
```

Comment:
Shall we introduce an alias type like ``FeatureId``? BTW we also have some type definitions for GeoJSON data here ``@here/harp-datasource-protocol/lib/GeoJsonDataType.ts`` and there it is still defined as ``id?: string;``

Reply:
GeoJsonDataType doesn't need to be changed, it just reflects the types according to the geojson standard. We just retain the number temporally for backwards compatibility with anyone that uses OMV data, cause we are now decoding numerical ids.

I don't think it's worth adding a FeatureId type since it'll be a simple string once we remove the backwards compatibility.

Reply:
OK